src/components/dataset/query/sidebar/AppliedFilterList.jsx

```diff
@@ -0,0 +1,166 @@
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { withStyles } from '@material-ui/core/styles';
+
+import Card from '@material-ui/core/Card';
+import List from '@material-ui/core/List';
+import ListSubheader from '@material-ui/core/ListSubheader';
+import ListItem from '@material-ui/core/ListItem';
+import { applyFilters } from '../../../../actions';
+import { Chip, Collapse, Badge } from '@material-ui/core';
+import { ExpandLess } from '@material-ui/icons';
+
+const styles = theme => ({
+  filterHeader: {
+    paddingTop: theme.spacing(1),
+    paddingBottom: theme.spacing(0.5),
+    lineHeight: 'inherit',
+  },
+  rangeInfo: {
+    display: 'inline-block',
+  },
+  inline: {
+    margin: '2px',
+  },
+  tableName: {
+    justifyContent: 'space-between',
+    fontWeight: 500,
+  },
+  box: {
+    margin: `${theme.spacing(1)}px ${theme.spacing(2)}px ${theme.spacing(1)}px ${theme.spacing(
+      2,
+    )}px`,
+  },
+});
+
+const StyledBadge = withStyles(theme => ({
+  badge: {
+    right: 12,
+    top: 0,
+    backgroundColor: theme.palette.primary.lightContrast,
+  },
+}))(Badge);
+
+export class AppliedFilterList extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: true,
+    };
+  }
+
+  static propTypes = {
+    classes: PropTypes.object,
+    filterData: PropTypes.object,
+    dataset: PropTypes.object,
+    dispatch: PropTypes.func.isRequired,
+    selected: PropTypes.string,
+    table: PropTypes.string,
+  };
+
+  handleExpand = () => {
+    const { open } = this.state;
+    this.setState({ open: !open });
+  };
+
+  clearFilter = (table, filter, datum) => {
+    const { dispatch, filterData, dataset, selected } = this.props;
+    const { relationships } = dataset.schema;
+    const clonedData = _.cloneDeep(filterData);
+    const clonedFilter = clonedData[table][filter];
+    const filterValue = clonedFilter.value;
+
+    if (clonedFilter.type !== 'range') {
+      if (Array.isArray(filterValue)) {
+        filterValue.splice(filterValue.indexOf(datum), 1);
+      }
+      if (_.isPlainObject(filterValue)) {
+        delete filterValue[datum];
+      }
+    }
+
+    if (_.isEmpty(filterValue) || clonedFilter.type === 'range') {
+      delete clonedData[table][filter];
+    }
+
+    if (_.isEmpty(clonedData[table])) {
+      delete clonedData[table];
+    }
+
+    dispatch(applyFilters(clonedData, relationships, selected, dataset));
+  };
+
+  render() {
+    const { classes, filterData, table } = this.props;
+    const { open } = this.state;
+    let numFilters = 0;
+    const listFilters = _.keys(filterData[table]).map(filter => {
+      const data = _.get(filterData[table], filter);
+      let dataString = data.value;
+
+      if (data.type === 'range') {
+        dataString = (
+          <Chip
+            onDelete={() => this.clearFilter(table, filter)}
+            className={classes.inline}
+            label={_.join(data.value, ' \u2013 ')}
```

Comment:
I had \u2013 pulled out as a constant named emDash which might have gotten lost in the merge conflicts, it might be a good idea to do that again here so it's clearer what the unicode is for